readthedocs/api/v2/views/footer_views.py

```diff
@@ -210,6 +211,14 @@ def get(self, request, format=None):
             'version_supported': version.supported,
         }
 
+        # increase the page view count
+        page_slug = request.GET.get('page', 'index')
```

Comment:
This should likely not default to `index`, no? If we don't have a `page`, we probably shouldn't be storing anything, or some other string like `rtd-unknown-page`, not `index`.

Reply:
I haven't seen any real examples where the page parameter is blank. It is `index` for the root and even with the dirhtml builder it uses the subdirectory name. I'll add a simple `if page_slug` check.